VERSION

```diff
@@ -0,0 +1 @@
+$Id$
```

Comment:
I'm not the biggest fan of this. I feel it's going to be a chore to not accidentally commit changes to this file.

Is this even needed?

Reply:
Ah, apparently this is a moot point. You can't commit the `$Id: ... $` changeset. 😅